packages/core/admin/admin/src/translations/sv.json

```diff
@@ -79,52 +79,70 @@
   "Roles.RoleRow.user-count": "användare",
   "Roles.components.List.empty.withSearch": "Det finns inga roller som matchar sökningen ({search})...",
   "Settings.PageTitle": "Inställningar - {name}",
-  "Settings.apiTokens.addFirstToken": "Lägg till ditt första API-token",
-  "Settings.apiTokens.addNewToken": "Lägg till ny API-token",
+  "Settings.apiTokens.addFirstToken": "Lägg till din första API-token",
+  "Settings.apiTokens.addNewToken": "Lägg till en ny API-token",
   "Settings.apiTokens.copy.editMessage": "Av säkerhetsskäl kan du bara se din token en gång.",
-  "Settings.apiTokens.copy.editTitle": "Denna token är inte tillgänglig längre.",
+  "Settings.apiTokens.copy.editTitle": "Denna token är inte längre tillgänglig.",
   "Settings.apiTokens.copy.lastWarning": "Kopiera denna token nu, den kommer inte visas igen!",
   "Settings.apiTokens.create": "Skapa en ny API-token",
   "Settings.apiTokens.description": "Lista över genererade tokens för att använda API:t",
   "Settings.apiTokens.emptyStateLayout": "Det finns inget innehåll ännu...",
-  "Settings.apiTokens.notification.copied": "Token har kopierats till urklipp.",
+  "Settings.apiTokens.ListView.headers.name": "Name",
+  "Settings.apiTokens.ListView.headers.description": "Description",
+  "Settings.apiTokens.ListView.headers.type": "Token type",
+  "Settings.apiTokens.ListView.headers.createdAt": "Created at",
+  "Settings.apiTokens.ListView.headers.lastUsedAt": "Last used",
+  "Settings.apiTokens.notification.copied": "Token har kopierats.",
   "Settings.apiTokens.title": "API-tokens",
   "Settings.apiTokens.types.full-access": "Full åtkomst",
-  "Settings.apiTokens.types.read-only": "Endast läsa",
+  "Settings.apiTokens.types.read-only": "Skrivskyddad",
+  "Settings.apiTokens.duration.7-days": "7 days",
+  "Settings.apiTokens.duration.30-days": "30 days",
+  "Settings.apiTokens.duration.90-days": "90 days",
+  "Settings.apiTokens.duration.unlimited": "Unlimited",
+  "Settings.apiTokens.form.duration": "Token duration",
+  "Settings.apiTokens.form.type": "Token type",
+  "Settings.apiTokens.duration.expiration-date": "Expiration date",
+  "Settings.apiTokens.createPage.permissions.title": "Permissions",
+  "Settings.apiTokens.createPage.permissions.description": "Only actions bound by a route are listed below.",
+  "Settings.apiTokens.RegenerateDialog.title": "Regenerate token",
+  "Settings.apiTokens.popUpWarning.message": "Are you sure you want to regenerate this token?",
```

Comment:
out of curiosity, some translations like this one seem to be in English - is it intended or is it an oversight? 🙂

Reply:
No that's not intended, will fix :+1:

Reply:
Fixed as of c369330